tailscale/config.yaml

```diff
@@ -21,6 +21,10 @@ hassio_api: true
 host_network: true
 map:
   - share:rw
+ports:
+  1080/tcp: 1080
+ports_description:
+  1080/tcp: SOCKS5 and HTTP proxy port for outbound connections into your Tailscale network (make empty to disable)
```

Comment:
Port descriptions should be in the translation files (this works, but is old-school).